src/main/resources/plugin.yml

```diff
@@ -5,19 +5,11 @@ api-version: ${project.bukkitAPIVersion}
 author: [Goosius,LlmDl]
 website: 'townyadvanced.github.io'
 prefix: ${project.artifactId}
-depend: [Towny]
+depend: [Towny, TownyChat]

```

Comment:
I wonder if this is something that should be added as a softdepend, and then have the listener only used when TownyChat is detected. There's probably some people out there that use other chat plugins, or don't want TownyChat and only want the cultures without the chat.